test/passing/skip.ml

```diff
@@ -93,14 +93,16 @@ let _ = (x [@ocamlformat "disable"]  [@test? _ when e [@test 2]]) 3
 let _ =
   let module X = struct
     let x = 4
-  end in
+  end
+  in
```

Comment:
Do you consider this an improvement? Or is this change to recover a bit from the effects of `disable` in the cases below?

Reply:
It looks more consistent with other `let-in` where `in` fall on its own line in case of a break.

Reply:
To be clear, the change about `let module M = struct end in` was not the target of that PR. Maybe I should restrict the scope and special case to remove this side-effect.

Reply:
Ok, then I am not sure of the intended change, can you point to an example?

Reply:
well, the regtest `module.ml` that was removed from failing and that I forgot add into passing :)

Reply:
Ok, let me look